hail/python/hail/vds/variant_dataset.py

```diff
@@ -173,11 +184,11 @@ def from_merged_representation(mt, *, ref_block_fields=(), infer_ref_block_field
 
         rmt = mt.filter_entries(
             hl.case()
-            .when(hl.is_missing(mt.END), False)
-            .when(hl.is_defined(mt.END) & mt[gt_field].is_hom_ref(), True)
+            .when(hl.is_missing(mt[ref_block_indicator_field]), False)
+            .when(hl.is_defined(mt[ref_block_indicator_field]) & mt[gt_field].is_hom_ref(), True)
             .or_error(
                 hl.str(
-                    'cannot create VDS from merged representation -' ' found END field with non-reference genotype at '
```

Comment:
TIL python lets you concatenate strings by just putting them next to each other like this 🤯